frontend/src/components/explorer/mapViewer/SidebarDataPanels.vue

```diff
@@ -7,7 +7,65 @@
           <i>{{ currentMap.name }}</i>
         </p>
       </header>
-      <footer v-if="currentMap.type !== 'custom'" class="card-footer">
+      <div v-if="currentMap.reactionList && missingReactionList.length > 0"
+           class="card-content p-4 sidebarCardHover">
+        <div class="content mb-0" @click="showModal = true">
+          Please note that {{ missingReactionList.length }}
+          of the reactions in the model are not shown on the map
+          <span class="icon"><i class="fa fa-info-circle"></i></span>
```

Comment:
This icon is not showing up for me
<img width="350" alt="image" src="https://user-images.githubusercontent.com/23480589/120190248-9f25f280-c218-11eb-90e8-ed043a6b975f.png">

Reply:
Weird, I tried on both Firefox and Chrome and it shows. Not sure why it wouldn't on yours. @e0 do you have any idea?

Reply:
It shows up for me too! Maybe try clearing your cache @mihai-sysbio

<img width="299" alt="image" src="https://user-images.githubusercontent.com/423498/120326035-91d83900-c2e8-11eb-908b-4d2f19571a6b.png">

Reply:
It did show up after a rebuild. Thanks for checking!